lib/import-export-visitor.js

```diff
@@ -738,9 +740,9 @@ function toModuleExport(specifierMap) {
 
     parts.push(
       exported,
-      ":()=>",
+      ":function(){return ",
```

Comment:
It looks like I can get away with not doing it for `module.export`. Do you want to split it?